File: 1.Fundamental-Level/5.Java-Basics/02.Java-Syntax/Java-Syntax-Homework/src/TriangleArea.java
import java.util.Scanner;


public class TriangleArea {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		System.out.print("Enter the coordinates of A: ");
		int Ax = scanner.nextInt();
		int Ay = scanner.nextInt();
		System.out.print("Enter the coordinates of B: ");
		int Bx = scanner.nextInt();
		int By = scanner.nextInt();		
		System.out.print("Enter the coordinates of C: ");
		int Cx = scanner.nextInt();
		int Cy = scanner.nextInt();
		
		int area = Math.abs((Ax * (By - Cy) + Bx * (Cy - Ay) + Cx * (Ay - By)) / 2);
		

		System.out.println("The area is: " + area);

	}
}

File: JavaScript-Development/JavaScript-Fundamentals/02.Control-Flow-Logic/05.TriangleArea/triangle-area.js
function triangleArea(args) {
    let a = Number(args[0]);
    let b = Number(args[1]);
    let c = Number(args[2]);
    let p = (a + b + c) / 2;
    let area = Math.sqrt(p * (p - a) * (p - b) * (p - c));
    console.log(area);
}

triangleArea([2, 3.5, 4]);